docs/environment.yml

```
@@ -6,7 +6,7 @@ channels:
   - numba/label/dev
 dependencies:
   - python=3.7
```

```suggestion
  - python=3.7
  - pip
```
Pip is missing as noted by `conda`, not sure it's related to the build problem but could well be not helping?

---

I have made the change but it did not alter any of the described symptoms.